cc_core.c

```diff
@@ -789,6 +789,12 @@ void emit_pop(int reg, char* note)
 		emit_out("\n");
 	}
 }
+
+int type_is_pointer(struct type* type_size)
```

Comment:
I find type_size a bit confusing... Maybe `current_type` ?

Reply:
`type_size` is used almost everywhere else. I'm not sure if it's because it previously only contained the size or if it's because (the old?) cc_* compilers couldn't handle a variable with the same name as the struct?

I agree that it doesn't really make sense but I'm using it for consistency.